content/docs/user-guide/project-config/_index.md

```diff
@@ -1,13 +1,42 @@
+
 | Topic | Description |
-| - | - | 
-| [Adding and Removing Gems](add-remove-gems/) | Learn how to add and remove Gems in your project. |
+| - | - |
+| [Understanding project.json](#understanding-project-json) | A detailed look at the fields in the project manifest, an important project configuration file. |
+| [Adding and Removing Gems](add-remove-gems/) | Learn how to add and remove Gems in your project. | 
 | [Registering Gems](register-gems/) | Learn how to register external Gems from sources outside of O3DE. |
 | [O3DE CLI Reference](cli-reference/) | A command line interface (CLI) reference for the `o3de` Python script.
 | [Troubleshooting](troubleshooting/) | Troubleshoot common issues that you might encounter during project configuration. |
+
+To build projects, you can use either Project Manager or **CMake**. You can find information on building projects in the [introductory tutorials](/docs/welcome-guide/create/) in the Getting Started Guide, and in the [Configure and Build](/docs/user-guide/build/configure-and-build) topic in the Build section of this User Guide.
+
+## Understanding `project.json` {#understanding-project-json}
+
+Each project contains a project manifest file named `project.json`. You can find this file in the project root directory. It stores important configuration properties for each project. It is created for you at the time of project creation.
+
+To make changes to any of the properties in this file, you can either edit it manually using a text editor, or specify individual properties to edit using the [O3DE CLI](./cli-reference) `edit-project-properties` command. Additionally, you can edit the `display_name` property in Project Manager.
+
+The following table describes each of the properties in `project.json`. The default value for many of the properties contains the name of the project given when the project was created. In the values below this is indicated by "<PROJECT_NAME>".
+
+| Property | Description | Default |
+| --- | --- | --- |
+| **project_name** | The name of the project. Used by the O3DE CLI `--project-name` parameter to identify the project. | "<PROJECT_NAME>" |
+| **origin** | A URL for the project, such as the repository URL or project website. This field is intended to be available for display and use by Project Manager. | "The primary repo for <PROJECT_NAME> goes here: i.e. http://www.mydomain.com" |
+| **license** | The license your project uses, and any copyright information you want to include. Consider providing the URL for the license. This field is intended to be available for display and use by Project Manager. | "What license <PROJECT_NAME> uses goes here: i.e. https://opensource.org/licenses/MIT" |
+| **display_name** | The display name of the project in Project Manager. | "<PROJECT_NAME>" |
+| **summary** | A short description of your project. This field is intended to be available for display and use by Project Manager. | "A short description of <PROJECT_NAME>." |
```

Comment:
```suggestion
| **summary** | A short description of the project. This field is intended for display and for Project Manager to use. | "A short description of <PROJECT_NAME>." |
```
Changed "your project" to "the project" for consistency with most other descriptions in the table. (Using "your" instead of "the" is totally fine; just be consistent.)